drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2540,10 +2542,11 @@ void RasterizerSceneGLES3::render_scene(const Ref<RenderSceneBuffers> &p_render_
 		glClear(GL_DEPTH_BUFFER_BIT);
 	}
 
-	if (!keep_color && !draw_feed) {
+	if (!keep_color) {
```

Comment:
This clear should happen _only_ when using an intermediate framebuffer. By default 2D and 3D rendering use the same texture. Therefore, this change will clear out the results of 2D rendering before 3D rendering happens. 

This PR works when using glow because the bound framebuffer uses a separate texture.

Reply:
Ah ok, so something more along the lines of:
```suggestion
	if (fbo != rt->fbo) {
		if (!keep_color) {
			clear_color.a = render_data.transparent_bg ? 0.0f : 1.0f;
			glClearBufferfv(GL_COLOR, 0, clear_color.components);
		}
		if (copy_framebuffer) {
			// Need to copy our current contents to our intermediate/MSAA buffer
			GLES3::CopyEffects *copy_effects = GLES3::CopyEffects::get_singleton();
			...
		}
	}
```

Reply:
Nice catch - updated.

I've also added a comparison matrix to the original PR description to show the different paths - it looks like `ENV_BG_KEEP` clears the 2D framebuffer in Forward+, do we know what the intended behaviour should be there?